### ISSUE TITLE
Remove this repo?

### ISSUE THREAD

**Issue by crhym3** [googlecodelabs/cloud-firebase#2]
@hostirosti @marcacohen I think this is superseded by https://github.com/firebase/friendlychat, right? In which case should we just remove this repo?

**Comment by marcacohen** [comment]
That's correct. @nicolasgarnier  asked me to remove the superseded codelab, which I did, but I forgot to clean up this repo. Thank for catching it. Deleting now...
